python/kfserving/kfserving/kfserver.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2019 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tornado.ioloop
+import tornado.web
+import tornado.httpserver
+import argparse
+import os
+import logging
+import json
+from typing import List, Dict
+from kfserving.handlers.http import PredictHandler, ExplainHandler
+from kfserving import KFModel
+
+DEFAULT_HTTP_PORT = 8080
+DEFAULT_GRPC_PORT = 8081
+
+parser = argparse.ArgumentParser(add_help=False)
+parser.add_argument('--http_port', default=DEFAULT_HTTP_PORT, type=int,
+                    help='The HTTP Port listened to by the model server.')
+parser.add_argument('--grpc_port', default=DEFAULT_GRPC_PORT, type=int,
+                    help='The GRPC Port listened to by the model server.')
+args, _ = parser.parse_known_args()
+
+KFSERVER_LOGLEVEL = os.environ.get('KFSERVER_LOGLEVEL', 'INFO').upper()
```

Comment:
is that better to set LOGLEVEL for kfserving SDK both for server and client?

Reply:
How about KFSERVING_LOGLEVEL

Reply:
Sounds great! shall we define this in the `constants.py` or a attribute in `__init__.py`？ I pefer in constants.